manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -286,13 +297,21 @@ For this step you should have ``Products.PDBDebugMode`` installed. It will make
 
     $ ./bin/test --all -s collective.package
 
-Remember that you can run ``./bin/test -s collective.package -D`` to find yourself in a pdb when a error happens.
+Remember that you can run ``./bin/test -s collective.package -D`` to enter a pdb session when an error occurs.
 
-With some luck, there are not too many issues with the code left at this point.
-
-It you are unlucky you have to fix Doctests. These should be changed so that Python 3 is the default. (e.g. text is represented as ``'foo'`` and not ``u'foo'`` and bytes are represented as ``b'bar'`` and not as ``'bar'``). Search for examples of ``Py23DocChecker`` in Plone's packages to find a pattern which allows updated doctests to pass in Python 2.
+With some luck, there will not be too many issues left with the code at this point.
 
+It you are unlucky then you have to fix Doctests.
+These should be changed so that Python 3 is the default.
+For example, string types (or text) should be represented as ``'foo'``, not ``u'foo'``, and bytes types (or data) should be represented as ``b'bar'``, not ``'bar'``.
+Search for examples of ``Py23DocChecker`` in Plone's packages to find a pattern which allows updated doctests to pass in Python 2.
 
+*   Test your code against `buildout.coredev on Jenkins <https://jenkins.plone.org/view/Add-ons/>`_.
+*   Note there are jobs set up for Plone 4.3, 5.1, and 5.2 on Python 2, and two jobs that run tests for Plone 5.2 on Python 3.6 and Python 3.7.
+*   Log in to the Jenkins website (top right) and click on the job you want to run.
+*   Choose the link "Build with parameters" in the left menu on the left-hand side.
+*   Fill the fields "ADDON_URL" and "ADDON_BRANCH" with your repository's URL and the branch name ("python3" if you followed these instructions).
+*   Start the build with the "Build" button.
 
```

Comment:
Maybe we should add a picture?
This is nice because it gives the user a visual and also it makes reading the docs a better experience.

Reply:
One other thing, we need to add the "gui label" directive to highlit the buttons, this is how we do it in other parts of the docs, too.

Reply:
I think this audience—developers upgrading an add-on—can get by without a screenshot. I also do not wish to add more work of maintaining screenshots of UIs, especially of third party applications.

Reply:
`:guilabel:` also needs to be added to the [rst Style Guide](https://docs.plone.org/about/contributing/rst-styleguide.html).

Reply:
Should guilabel be added to all UI elements? Form controls, links, menu items, etc.?

Reply:
Good question, till now if I remember right, we use it for "actions" like buttons and other stuff the user has to "click" on.
There is no "rule" yet afaik.

Reply:
I found lots of guilabels for all UI stuff.

Updated in ccff345